packages/client/styles/theme/global.css

```diff
@@ -495,7 +497,16 @@
 .ProseMirror p {
   margin-block-start: 4px;
   margin-block-end: 4px;
-  line-height: 1rem;
+  /* Line height: for now most editable surfaces are 14/20 */
+  line-height: 1.4285714286em;
```

Comment:
Is `em` intended? That might get tricky when we add font sizes to tiptap. I did a little digging & saw inheritance gets tricky with units, but unitless is a little cleaner https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values

Reply:
Ah, good point! I can yank the `em` unit. In the future, I want to find a good way to set up the base font size as 16px for pages, and 14px for cards, comments, etc. but we shall see.